src/main/kotlin/no/nav/familie/ef/sak/journalføring/JournalpostUtil.kt

```diff
@@ -21,4 +21,4 @@ fun DokumentInfo.harOriginaldokument() =
 
 fun Journalpost.harUgyldigAvsenderMottaker(): Boolean = this.journalposttype != Journalposttype.N && this.avsenderMottaker?.navn.isNullOrBlank()
 
-fun Journalpost.manglerAvsenderMottaker(): Boolean = this.avsenderMottaker?.erLikBruker != true && this.avsenderMottaker?.navn.isNullOrBlank() || this.avsenderMottaker?.id.isNullOrBlank()
+fun Journalpost.manglerAvsenderMottaker(): Boolean = (this.avsenderMottaker?.erLikBruker != true && this.avsenderMottaker?.navn.isNullOrBlank()) || this.avsenderMottaker?.id.isNullOrBlank()
```

Comment:
Funksjonen #manglerAvsenderMottaker er testet og feiler ved parenteser rundt || 

_In Kotlin, the logical AND operator (&&) has higher precedence than the logical OR operator (||). This means that in an expression containing both && and || without parentheses, the && operations will be evaluated before the || operations._